### ISSUE TITLE
On Intro Monologue screen, scrolling down past the end of the monologue causes a crash

### ISSUE THREAD

**Issue by stephengtuggy** [vegastrike/Assets-Masters#30]
The error appears to be coming from the Python asset code:

```
TypeError: list indices must be integers or slices, not float
```

**Comment by P-D-E** [comment]
In `modules/GUI.py`, replacing line 1659
```
        self.firstVisible = max(0,min(len(self.items)-1-len(self._listitems)/2,self.firstVisible + lines))
```
with
```
        self.firstVisible = max(0,min(len(self.items)-1-len(self._listitems)//2,self.firstVisible + lines))
```
solves the issue, evaluating to 202 instead of 201.5 for the last intro page.

**Comment by stephengtuggy** [comment]
@P-D-E Cool! Thanks for figuring that out

**Comment by stephengtuggy** [comment]
@P-D-E Can you submit that fix as a pull request?

**Comment by P-D-E** [comment]
You're welcome, PR right above.

**Comment by stephengtuggy** [comment]
PR merged. Now we just need the same fix applied to the 0.8.x branch, and we'll be good to go.

**Comment by P-D-E** [comment]
@stephengtuggy Can you backport it or should I issue a separate PR on 0.8.x?

**Comment by stephengtuggy** [comment]
> @stephengtuggy Can you backport it or should I issue a separate PR on 0.8.x?

@P-D-E Please do.

**Comment by P-D-E** [comment]
@stephengtuggy Done, PR above.

**Comment by stephengtuggy** [comment]
Issue fixed for both 0.8.x and master. Closing.
